project/plugins.sbt

```diff
@@ -17,7 +17,7 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.3")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-s3" % "0.5")
 
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.6")
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.12")
```

Comment:
Actually, just bumped to `0.1.14` as today's `1.8` JMH release has an interesting new profiler (not for this benchmark, but in general interesting): http://mail.openjdk.java.net/pipermail/jmh-dev/2015-April/001785.html (gc churn)